visualizer/script.js

```diff
@@ -305,11 +305,46 @@ fetch(`../spoiler.json`).then(c => c.json()).then(c => {
 	}
 	gen_obscurity(null);
 	let el = document.getElementById("room-info");
-	let dragging = false;
-	let dragged = false;
-	document.getElementById("map").ondragstart = ev => {
+	let dragged = false, dragging = false;
+	var scale = 1, page_x = 0, page_y = 0;
+	let m = document.getElementById("map");
+	m.ondragstart = ev => {
 		return false;
 	}
+	m.ondragstart = ev => {
+		return false;
+	}
+	m.onmousedown = ev => {
+		dragging = true;
+	}
+	m.onwheel = ev => {
+		const scaleOld = scale;
+		var z = document.getElementById("zoom");
+
+		scale += ev.deltaY * -0.001;
```

Comment:
```suggestion
		scale *= 1.0 - ev.deltaY * 0.0005;
```
By multiplying to change the scale instead of adding, I think it can help give a consistent feel across different scales. Otherwise I think it feels more sensitive when zoomed out compared to zoomed in.